rcaracaus/pledge-vegan-node
# Removed search feature until it works.

Steps to test:

1. Go to localhost:3000
2. Check that there is no search button in the top right of the home page or any other page
